src/main/scala/mesosphere/marathon/api/v2/GroupsResource.scala

```diff
@@ -243,7 +243,7 @@ class GroupsResource @Inject() (
       val raw = Json.parse(body).as[raml.GroupUpdate]
       val effectivePath = raw.id.map(id => validateOrThrow(PathId(id)).canonicalPath(rootPath)).getOrElse(rootPath)
 
-      val groupValidator = Group.validNestedGroupUpdateWithBase(effectivePath, originalRootGroup)
+      val groupValidator = Group.validNestedGroupUpdateWithBase(effectivePath, originalRootGroup, Group.updateModifiesServices(raw))
       val groupUpdate = validateOrThrow(
         GroupNormalization(config, originalRootGroup).updateNormalization(effectivePath).normalized(raw)
       )(groupValidator)
```

Comment:
On thing to keep in mind here, is that the PATCH doesn't create a deployment, the PUT does. Keeping the enforceRole-Updates out of Deployments sounds like a good thing to me...